postgres/datadog_checks/postgres/postgres.py

```diff
@@ -44,64 +45,27 @@
 from .version_utils import V8_3, V9, V9_1, V9_2, V9_4, V9_6, V10, get_raw_version, parse_version, transform_version
 
 MAX_CUSTOM_RESULTS = 100
-TABLE_COUNT_LIMIT = 200
```

Comment:
(nit) Constants could be in a `constants.py` file to avoid having them in multiple places.

Reply:
I think that's only worth it if they're shared (used in different places). I think its ok for constants to be in different places if they're only used in the place they're defined